{{ cookiecutter.repo_name }}/settings/codequality/quality.gradle

```diff
@@ -17,8 +17,8 @@ configurations {
 }
 dependencies {
     checkstyle 'com.puppycrawl.tools:checkstyle:8.39'
-    ktlint "com.pinterest:ktlint:0.40.0"
-    detekt 'io.gitlab.arturbosch.detekt:detekt-cli:1.15.0'
+    ktlint "com.pinterest:ktlint:0.47.0"
+    detekt 'io.gitlab.arturbosch.detekt:detekt-cli:1.17.0-RC2'
```

Comment:
I would rather suggest to use stable version `1.21.0`. Any special reason behind using `1.17.0-RC2` ? It's showing `1.21.0` as [latest stable](https://mvnrepository.com/artifact/io.gitlab.arturbosch.detekt/detekt-cli) version.

Reply:
actually we are getting Property 'formatting' is misspelled or does not exist and i will try all new version but not worked and find only workaround to update this 1.17.0-RC2

Reply:
Sorry I didn't get you @Jitendrasimform

Reply:
@prem-p-simform i m getting Property 'formatting' is misspelled or does not exist. issue when i use any updated detekt lib , and it's working with 1.17.0-RC2 and also doing googling regard this issue but not find out

Reply:
Resolve detekt new version with detekt plugin with formating library added